SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/CrossServerAbilityActivationTest/SpyValueGASTestActor.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "../TestActors/GASTestActorBase.h"
```

Comment:
Better fix this relative include path

Reply:
The only alternative is a full module include path (i.e. "SpatialGDKFunctionalTests/SpatialGDK/..."). I picked the relative path for being shorter.

Reply:
Alright it's back to the full include.